modules/ROOT/pages/proxy-deploy-runtime-fabric.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-include::1.0@runtime-fabric:ROOT:page$proxy-deploy-runtime-fabric.adoc[tag=content]
+include::runtime-fabric:ROOT:page$proxy-deploy-runtime-fabric.adoc[tag=content]
```

Comment:
This works without a version?  Interesting!

Reply:
It does. When no version is specified, it automatically uses the latest available.